web/src/util/index.js

```diff
@@ -161,3 +161,6 @@ export const arrToObj = (array = [], initialValue = 0) =>
 export const addObjVals = obj => Object.values(obj).reduce((a, b) => a + b, 0);
 
 export const titleCase = str => str.replace(/\b\S/g, t => t.toUpperCase());
+
+export const isProgamAdmin = activity =>
+  activity.name === 'Program Administration' || activity.id === 1;
```

Comment:
I'm wondering if `isKey` should be a property on the activity state object.  `key` could be defined as the activity with the smallest ID (since the IDs will change over time because of the way we're kind of abusing the database 😬).  Then you could pull it off the activity state in all those components instead of asking a helper?

Got me further wondering if `isKey` should be in the data model as well.  At some point we probably ought to change the API behavior to not delete and re-add everything every time, but once we do that, we can't guarantee ordering anymore (and we should probably address that, too, but I've got an idea about that).

Anyway...